backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -141,7 +141,7 @@ extern "C" {
         gpu_count: u32,
         lwe_array_out: *mut c_void,
         glwe_in: *const c_void,
-        indexes_array: *const c_void,
+        indexes_array: *const u32,
```

Comment:
I think the c++ entry point should be modified as well if you change this, right now it takes `void*`. We should always pass u32 for indexes arrays.